config/prow/config_knative.yaml

```diff
@@ -433,6 +433,39 @@ periodics:
           memory: 12Gi
         limits:
           memory: 16Gi
+    - dot-release: true
+      dot-dev: true
+      go113: true
+      release: "0.10"
+      env-vars:
+      - RELEASE_BRANCH=releaes-0.10
+      resources:
+        requests:
+          memory: 12Gi
+        limits:
+          memory: 16Gi
+    - dot-release: true
+      dot-dev: true
+      go113: true
+      release: "0.11"
+      env-vars:
+      - RELEASE_BRANCH=releaes-0.11
+      resources:
+        requests:
+          memory: 12Gi
+        limits:
+          memory: 16Gi
+    - dot-release: true
+      dot-dev: true
+      go113: true
+      release: "0.12"
+      env-vars:
+      - RELEASE_BRANCH=releaes-0.12
+      resources:
+        requests:
+          memory: 12Gi
+        limits:
+          memory: 16Gi
     - dot-release: true
```

Comment:
Let's also change this config by adding the release field, to make them consistent?

Reply:
you mean 13?

Reply:
Yes. Then it will be similar as how we configure the periodic jobs for the release branches.

Reply:
discussed offline, we agreed that this is expected